.github/dependabot.yml

```diff
@@ -46,15 +94,15 @@ updates:
     directory: "/internal/buildscripts/packaging/msi/msi-builder"
     schedule:
       interval: "weekly"
-  - package-ecosystem: "github-actions"
```

Comment:
Looks like this entry was missed in Makefile , fixing https://github.com/signalfx/splunk-otel-collector/pull/2721